lib/apis/moray.js

```diff
@@ -586,8 +596,25 @@ Moray.prototype.putVm = function (uuid, vm, cb) {
     this.connection.putObject(VMS_BUCKET_NAME, uuid, object, cb);
 };
 
+/*
+ * Deletes *all* VMs that match the filter generated from "params".
+ */
```

Comment:
Right, agreed. I'll move it to `test/lib` because in its current form it would be too easy to use outside of tests. Thanks!